.github/workflows/build.yml

```diff
@@ -47,7 +47,6 @@ env:
   NX_CACHE_RESTORE_KEYS: |
     nx-Linux-${{ github.ref }}-${{ github.event.inputs.commit || github.sha }}
     nx-Linux-${{ github.ref }}
-    nx-Linux-refs/heads/develop
```

Comment:
Note: I figured this isn't actually necessary, because the way cache works is that it _only_ ever fetches from the current branch OR the default branch (=develop), so it can't even get the cache from a different branch. so should be all good without this.